src/AutoRest.CSharp/Common/Output/Models/ClientFields.cs

```diff
@@ -109,6 +117,7 @@ private ClientFields(IEnumerable<Parameter> parameters, InputAuth? authorization
             _fields = fields;
             _parameterNamesToFields = parameterNamesToFields;
             CredentialFields = credentialFields;
+            ScopeDeclarations = new CodeWriterScopeDeclarations(fields.Select(f => f.Declaration));
```

Comment:
@archerzz This is where `ScopeDeclarations` is created to be used in scopes later